src/ussd_tb_check.js

```diff
@@ -484,6 +484,8 @@ go.app = function () {
               self.pad_location(location.lat, 2) +
               self.pad_location(location.lng, 3) +
               "/";
+            self.im.user.location_lat = location.lat;
```

Comment:
I don't see where `self.im.user.location_lat` is used anywhere after you set it here?